feelpp/feel/feelfilters/exporterensightgold_impl.hpp

```diff
@@ -125,101 +109,43 @@ ExporterEnsightGold<MeshType,N>::init()
     /* Init number of digit for maximum time step */
     M_timeExponent = 4;
 
-    /* if we do not want to merge the results from the different processes */
-    /* we isolate each process by using worldCommSeq() from Environment */
-    /* Each process will be in a group seeing only itself and not every */
-    /* process as worldComm() */
-    if( ! boption( _name = "exporter.ensightgold.merge.markers" ) )
-    {
-        //this->setWorldComm(Environment::worldCommSeq());
-        this->setWorldComm(this->worldComm().subWorldCommSeq());
-    }
+    M_mergeTimeSteps = boption( _name="exporter.ensightgold.merge.timesteps" );
```

Comment:
I know that these options were already used in the class but could we get them at the constructor time ? and use members to decide what to do here ? otherwise we can control the class only with command line options. that is not good